xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -425,7 +426,7 @@ namespace
 void GetItemsForPlayList(const std::shared_ptr<CFileItem>& item, CFileItemList& queuedItems)
 {
   if (VIDEO_UTILS::IsItemPlayable(*item))
-    VIDEO_UTILS::GetItemsForPlayList(item, queuedItems);
+    GetItemsForPlayList(item, queuedItems);
```

Comment:
Doesn't this now call itself recursively? I think the namespace needs to be kept to call the right function.

Reply:
yeh. bit too blind faith in clangd suggestions.